building_map_tools/building_crowdsim/test/build_navmesh_Base_TEST.py

```diff
@@ -0,0 +1,24 @@
+from building_crowdsim.navmesh.build_navmesh import BuildNavmesh
+
+
+def test():
```

Comment:
Same as the comment for the `build_configfile_TEST.py` I also wonder whether those tests are integrated with colcon so that they are ran when doing `colcon test`?